sdcm/utils/common.py

```diff
@@ -787,8 +798,16 @@ def terminate_runner_instance(backend, region, sct_runner):
         if not keep or seconds_running > keep_hours * 3600:
             LOGGER.info("[%s] Runner instance '%s'<keep=%s> that launched at '%s' UTC "
                         "is unused/expired, cleaning ...", region, instance_id, keep, launch_time)
-            terminate_runner_instance(backend, region, sct_runner)
+            terminate_runner_instance(aws_client=aws_client, backend=backend, region=region, sct_runner=sct_runner)
             runners_cleaned.append(sct_runner)
+
+        if test_status == "SUCCESS" and test_runner_ip:
```

Comment:
Is double termination a valid case here?

Reply:
the probability of such a situation is very low and the consequences of it are minor. Let's see if it happen to us and then decide.